change/@itwin-grouping-mapping-widget-ac9181be-f95d-4769-af73-e0490c6fcfb8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Why `minor`?

Reply:
Behavior change in the query building. Seems pretty breaking to me.